PenEvents/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~
+  ~  Copyright (c) Microsoft Corporation. All rights reserved.
+  ~  Licensed under the MIT License.
+  ~
+  -->
+
+<resources>
+    <string name="pen_events_app_name">Stiftereignisse</string>
```

Comment:
It doesn't have a lot of sense in German, and in other samples we have English names for the samples-names